packages/ui/react-ui-mosaic/src/mosaic/Tile.tsx

```diff
@@ -45,6 +45,8 @@ export type MosaicTileProps<TData extends MosaicDataItem = MosaicDataItem, TPosi
     onSelect?: () => void;
     onDelete?: (force?: boolean) => void;
     onNavigate?: () => void;
+    onAddBefore?: () => void;
+    onAddAfter?: () => void;
```

Comment:
🫣

Reply:
I know… it’s another part of this interface we may like to factor out. Mosaic and `app-framework` are increasingly seeming intertwined…